### PR TITLE
[Naitei] Search_User_Admin

### PR DESCRIPTION
**Pull request by ThaiKhacDuoc**
## Related Tickets
- ticket redmine

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)

https://github.com/user-attachments/assets/e0d7e016-9b5d-465d-95ed-65027b30507c

### CODE REVIEW

**Comment by ThaiKhacDuoc** [comment]
ready

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+                         items: Settings.page_size)
   end
 end
+
+private
+
+def search_params
+  q_params = params[:q] || {}
+  q_params[:created_at_gteq] = parse_date(q_params[:created_at_gteq])
+  q_params[:created_at_lteq] = parse_date(q_params[:created_at_lteq])
```
Comment (ghost) [inline app/controllers/admin/users_controller.rb]: Dùng ransacker đi em

**Review by ghost — COMMENTED** [review]
cmt

**Comment by ThaiKhacDuoc** [comment]
ready

**Comment by ThaiKhacDuoc** [comment]
ready

**Review by ghost — APPROVED** [review]

**Pull request merged**